src/components/App.jsx

```diff
@@ -1,7 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import QuestionsList from './QuestionsList';
+import AddQuestionBtn from './AddQuestionBtn';
+import getProductQuestions from '../api/getProductQuestions';
+import getProductInfo from '../api/getProductInfo';
 
 function App() {
+  const [productId, setProductId] = useState(3);
```

Comment:
i'm guessing this is preparing for a future time when the product Id is going to change with a search or something.  if that's the case, that makes sense to me, otherwise, I'm not sure why this needs to be a state var